lib/workload/stateless/sequence_run_manager/src/sequence_run_manager_proc/lambdas/bssh_event.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+import django
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "sequence_run_manager.settings.base")
```

Comment:
Do we need to set a default? 
Seems dangerous considering that the base config states it should not be used in production?

Reply:
Base as default is fine, Flo. In our deployment situation.
Those comments were a bit legacy, but good to know as dev note.